src/collections/_documentation/platforms/python/index.md

```diff
@@ -2,55 +2,398 @@
+Import and initialize the Sentry SDK early in your application’s setup:
 
-### Task Queues
+```python
+import sentry_sdk
+sentry_sdk.init("https://<key>@sentry.io/<project>")
+```
 
-* [Celery]({% link _documentation/platforms/python/celery.md %})
-* [RQ (Redis Queue)]({% link _documentation/platforms/python/rq.md %})
+### Verifying Your Setup
 
-### Serverless
+Great! Now that you’ve completed setting up the SDK, maybe you want to quickly test out how Sentry works. Start by capturing an exception:
 
-* [AWS Lambda]({% link _documentation/platforms/python/aws_lambda.md %})
-* [Generic Serverless]({% link _documentation/platforms/python/serverless.md %})
+```python
+sentry_sdk.capture_exception(Exception("This is my fake error message"))
+```
 
-### Other Integrations
+Once the exception is captured, you'll see the error in your Sentry dashboard.
 
-* [Logging]({% link _documentation/platforms/python/logging.md %})
-* [GNU Backtrace]({% link _documentation/platforms/python/gnu_backtrace.md %})
-* [Default integrations]({% link _documentation/platforms/python/default-integrations.md %})
+## Capturing Errors
 
-{% capture __alert_content -%}
-Looking for [the legacy Python SDK "Raven"?]({%- link _documentation/clients/python/index.md -%})
-{%- endcapture -%}
-{%- include components/alert.html
-  title="Note"
-  content=__alert_content
-%}
+In Python you can either capture a caught exception or the one currently held in `sys.exc_info()` by not passing an argument:
+
+```python
+from sentry_sdk import capture_exception
+
+try:
+    a_potentially_failing_function()
+except Exception as e:
+    # Alternatively the argument can be omitted
+    capture_exception(e)
+```
+
+### Capturing Messages
+
+Another common operation is to capture a bare message. A message is just some textual information that your app can send to Sentry. Typically, messages are not emitted, but there are situations when this is useful.
+
+```python
+from sentry_sdk import capture_message
+
+capture_message('Something went wrong')
+```
+
+## Releases
+
+A release is a version of your code that you deploy to an environment. When you give Sentry information about your releases, you unlock many new features:
+
+- Determine the issue and regressions introduced in a new release
+- Predict which commit caused an issue and who is likely responsible
+- Resolve issues by including the issue number in your commit message
+- Receive email notifications when your code gets deployed
+
+After configuring your SDK, setting up releases is a 2-step process:
+
+1. [Create Release and Associate Commits]({%- link _documentation/workflow/releases.md -%}#create-release)
+2. [Tell Sentry When You Deploy a Release]({%- link _documentation/workflow/releases.md -%}#create-deploy)
+
+For more information, see [Releases Are Better With Commits](https://blog.sentry.io/2017/05/01/release-commits.html).
+
+## Adding Context
+
+You can also set context when manually triggering events.
```

Comment:
I figured you could by the title, whats context? You explain context below but its under the guise of setting it (which isnt actually explained). Just kill the next header, kill this sentence.